sympy/core/tests/test_subs.py

```diff
@@ -8,6 +8,7 @@
 from sympy.core.basic import _aresame
 from sympy.utilities.pytest import XFAIL
 from sympy.abc import x, y, z
+from sympy.physics.units import kilogram, meter, second, gravitational_constant
```

Comment:
This test should be in sympy/physics/units/tests/test_quantities.py not here. Because the functionality added (custom substitution for Quantity) is implemented in that module, and tests go together with implementation. For example, Derivative and Integral class implement their own subs methods, but they are not tested in core/tests/test_subs - they are tested in those classes.